services/backend/src/routes/connectivity.js

```diff
@@ -114,6 +114,43 @@ module.exports = (errorHandler, connectivityService) => {
       });
   }));
 
+
+  /**
+   * @apiGroup s_pa_connectivity
+   * @api {get} /connectivity/timeline TimelineByCategory
+   * @apiName TimelineByCategory
+   * @apiVersion 1.0.0
+   * @apiDescription
+   * Values through time of a protected area connectivity category in a given area
+   *
+   * @apiParam (Query params) {String|Number} areaType area type
+   * @apiParam (Query params) {String} areaId area id
+   * @apiParam (Query params) {String} category category of connectivity index
+   *
+   * @apiSuccess {Object} result
+   * @apiSuccess {Object} result.key PA connectivity category
+   * @apiSuccess {Object} result.data timeline data values x (year) and y (percentage)
```

Comment:
result.data is an array of objects

Reply:
As result.key is a string then 131 also should be changed to:

@apiSuccess {String} result.key category of the connectivity index

And 132 will be changed to:
 
@apiSuccess {Object[]} result.data timeline data values x (year) and y (percentage)

it is ok?

Reply:
Yes and Yes

Reply:
Nop, you should describe x and y separately as result.data.x and result.data.y